### PR TITLE
git-var: add page

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #3953

### CODE REVIEW

File: pages/common/git-var.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# git var
+
+> Prints a Git logical variable.
```
Comment (sbrl) [inline pages/common/git-var.md:3]: Perhaps this will read better?
```suggestion
> Prints the value of a Git logical variable.
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left a comment below for you to review.

File: pages/common/git-var.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/git-var.md:3]: Yep.  I copied it right from https://git-scm.com/docs/git-var.

**Comment by schneiderl** [comment]
@bl-ue PR looks good but I have a question: in which context those commands that you have opened PRs for are used? It looks like most of the functionality can be achieved by using "git" and its subcommands.

**Review by schneiderl — APPROVED** [review]

File: pages/common/git-var.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# git var
+
+> Prints a Git logical variable's value.
+> More information: <https://git-scm.com/docs/git-var>.
+
+- Print the value of a Git logical variable:
+
+`git var {{GIT_AUTHOR_IDENT|GIT_COMMITTER_IDENT|GIT_EDITOR|GIT_PAGER}}`
+
+- List all Git logical variables (including those found in `.git/config`; deprecated in favor of `git config -l`):
```
Comment (navarroaxel) [inline pages/common/git-var.md:10]:
```suggestion
- [l]ist all Git logical variables (including those found in `.git/config`; deprecated in favor of `git config --list`):
```

File: pages/common/git-var.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`git var -l`
```
Comment (navarroaxel) [inline pages/common/git-var.md:12]: Why include a deprecated command?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/git-var.md (hunk 3 above)
Comment (bl-ue) [reply pages/common/git-var.md:12]: It's not deprecated, just the listing of the config variables is deprecated.  If you run `git var -l`, you'll notice the same output as `git config -l`, plus some `CONSTANT_CASE` variables at the end.  This command is still in Git and still works.

Comment (sbrl) [reply pages/common/git-var.md:12]: Oh, I see. So only half the command's functionality is deprecated?

**Review by sbrl — APPROVED** [review]
Looks ok to me after @navarroaxel's comment is addressed

File: pages/common/git-var.md (hunk 3 above)
Comment (bl-ue) [reply pages/common/git-var.md:12]: Yes.  Exactly.

Comment (navarroaxel) [reply pages/common/git-var.md:12]: Should we remove the example @sbrl ?

**Comment by bl-ue** [comment]
@schneiderl I didn't see your comment!

>@bl-ue PR looks good but I have a question: in which context those commands that you have opened PRs for are used? It looks like most of the functionality can be achieved by using "git" and its subcommands.

Yes, I know that, and it makes thing feel a little silly :D but I decided to keep doing it (1) just for the sake of completeness and (2) so that users can know a little about the command&mdash;and what to look for instead.

Do you agree?

File: pages/common/git-var.md (hunk 3 above)
Comment (bl-ue) [reply pages/common/git-var.md:13]: @sbrl @navarroaxel take a look at https://github.com/tldr-pages/tldr/commit/b680abe810d86b37a9cdbba0223cc2f2b2bac00d

**Comment by schneiderl** [comment]
@bl-ue Agreed. I don't think we are the ones who decide which commands our user will use. Our job is to have everything documented as good as possible. 😄 
Thank you for the job you've been doing here over the past weeks.

File: pages/common/git-var.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/git-var.md]: Actually I like this better:
```diff
-Prints a Git logical variable's value.
+Prints the value of a Git logical variable.
```
WDYT, @sbrl?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**